### PR TITLE
Add python release workflow and changelog

### LINKED ISSUE #741: [Task]: Create a workflow for pypi uploading.
## Description

Create a workflow for pypi uploading.

/cc @bingyanglin

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Add python release workflow and changelog

## Links to any relevant issues

Fixes https://github.com/iotaledger/wallet.rs/issues/741

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by Thoralf-M** [comment]
Release would currently fail on macos because of rocksdb, see https://github.com/Thoralf-M/wallet.rs/actions/runs/3695378920/jobs/6257680734

File: bindings/python/setup.py [hunk 1]
```diff
@@ -22,10 +22,10 @@ def get_py_version_cfgs():
 
 setup(
     name="iota_wallet",
-    version="0.2.0",
+    version="1.0.0.rc.1",
     classifiers=[
         "License :: SPDX-License-Identifier ::  Apache-2.0",
-        "Development Status :: 0.1.0 - Alpha",
+        "Development Status :: 1.0.0.rc.1",
```
Comment (thibault-martinez) [inline bindings/python/setup.py]:
```suggestion
        "Development Status :: 1.0.0-rc.1",
```
?

Comment (thibault-martinez) [reply bindings/python/setup.py]: Or maybe python doesn't like that ?

Comment (Thoralf-M) [reply bindings/python/setup.py]: Makes sense, will also update in iota.rs

**Review by thibault-martinez — DISMISSED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Review by rajivshah3 — APPROVED** [review]

**Pull request merged**